qemu/tests/cfg/guest_iommu_test.cfg

```diff
@@ -38,3 +38,16 @@
             only Linux
             enable_guest_iommu = yes
             guest_iommu_option = pt
+    variants diff_parameter:
+        - eim_off:
+            iommu_eim = off
+            check_key_words = "DMAR-IR: dmar0 does not support EIM;"
+            check_key_words += "DMAR-IR: Enabled IRQ remapping in xapic mode"
+        - eim_on:
+            iommu_eim = on
+            check_key_words = "DMAR-IR: Queued invalidation will be enabled to support x2apic and Intr-remapping;"
+            check_key_words += "DMAR-IR: Enabled IRQ remapping in x2apic mode"
+        - with_more_vcpus:
+            vcpu_maxcpus = 10
```

Comment:
Can not take for granted that it is "more vcpu", because you don't define a default one for other cases.